README_TTN.md

```diff
@@ -10,70 +10,84 @@ Official tutorial: https://www.thethingsindustries.com/docs/gateways/thethingski
 
 ```
 GatewayID:
-ttn-martel-innovate-001@ttn
+martel-ttn-01
 
 Account Server:
 https://eu1.cloud.thethings.network/
 
 Gateway Key:
-NNSXS.V2QQSQL73SK5CRMVTPHS24PMIPMXOC6ZIKZS7DY.H7SFIQH2FNHITEJBELFZKKNVPZQRIF5Z22N7LE4GWAA5WX3IZJ2Q
+NNSXS.NACIYY37UADMZG6O6XD7HRH6OM6XDHEKEIUU4CI.VSZHH6U77GJZHBQB2LNO6RQN3RFONSVYEUKI3OBZXPJGCU5EB4KQC0MM3NT01T4L1ANO
 ```
+- Ricordare di rimuovere la parte di sicurezza. 
```

Comment:
english, thx :)

Reply:
Giacomo had asked me to write it in Italian, I can modify it if you prefer.